Makefile

```diff
@@ -311,6 +311,7 @@ endef
 lint: export TARGET := clojure
 lint: ##@test Run code style checks
 	@sh scripts/lint-re-frame-in-quo-components.sh && \
+	sh scripts/lint-old-quo-usage.sh && \
```

Comment:
Is there any good reason why we are doing all of this silly `&& \` stuff? The only place I can think of is where `ALL_CLOJURE_FILES` is used, but that can be don in-line.

Reply:
I just copied the code without paying much attention 😬. I've now corrected the line I added.

It seems there are multiple instances where we've used `&& \` where it may not be necessary. I think it's better to address this in a separate PR. @jakubgs, wdyt?

Reply:
That fair.